server/controllers/projects/retrieveProject.js

```diff
@@ -1,9 +1,11 @@
 /**
- * @api {GET} /projects Retrieve projects
+ * @api {GET} /projects Retrieve project
  * @apiName Retrieve projects
  * @apiGroup Projects
  * @apiVersion 1.0.0
  *
+ * @apiHeader {String} projectId Id of the project to be retrieved.
```

Comment:
We send the projectId through path not header, right?

Reply:
Right, fixed!